pep-8001.rst

```diff
@@ -97,7 +97,76 @@ The election will be supervised by Ernest W. Durbin III, The PSF Director of Inf
 The results of the election, including anonymized ballots, will be made public on
 December 17th, after the election has closed.
 
-The following settings will be used for the vote in the CIVS system::
+The following settings will be used for the vote in the CIVS system:
+
+Name of the poll: ``Python governance vote (December 2018)``
+
+Description of the poll::
+
+    This is the vote to choose how the CPython project will govern
+    itself, now that Guido has announced his retirement as BDFL. For
+    full details, see <a
+    href="https://www.python.org/dev/peps/pep-8001/">PEP
+    8001</a>. Many discussions have occurred under <a
+    href="https://discuss.python.org/tags/governance">the "governance"
+    tag</a> on discuss.python.org.
+    <p>
+    All votes must be received <b>by the end of December 16th, 2018, <a
+    href="https://en.wikipedia.org/wiki/Anywhere_on_Earth">Anywhere on
+    Earth</a></b>. All CPython core developers are eligible to vote.
+    <p>
+    <b>Note: You can only vote once, and all votes are final.</b> Once
+    you click "Submit ranking", it's too late to change your mind.
+    <p>
+    All ballots will be published at the end of voting, but <b>without
+    any names attached</b>. No-one associated with the Python project or
+    the PSF will know how you voted, or even whether you voted.
+    <p>
+    If you have any questions, you can post in <a
+    href="https://discuss.python.org/c/committers">the Committers
+    topic</a>, on <a href="mailto:python-committers@python.org">the
+    python-committers list</a>, or <a
+    href="mailto:ewdurbin@gmail.com">contact the vote administrator
+    directly</a>.
+    <p>
+    <h1>Options</h1>
+    <p>
+    We're selecting between seven PEPs, each proposing a different
+    governance model.
+    <p>
+    The options below include links to the text of each PEP, as well
+    as their complete change history. The text of these PEPs was
+    frozen on December 1, when the vote started. But if you looked at
+    the PEPs before that, they might have changed. For convenience,
+    the following is a complete list of changes since the official
+    review period started on November 16:
+    <p>
+    <ul>
+      <li><i>(no changes)</i></li>
```

Comment:
In the event that a change appears between now and December 1st AoE time (7AM tomorrow morning for me here in Eastern Time). What should be put here?

Reply:
I don't think this summary is necessary as keeping it in-sync will be a hassle as shown by Ernest's question (e.g. there's already a couple of PRs waiting to be merged). I say people look at the git history of a PEP to see how it's changed since the last time someone read a PEP.